concent_api/core/tests/test_payments_bankster.py

```diff
@@ -87,3 +93,192 @@ def test_that_claim_deposit_return_true_and_false_if_provider_has_less_funds_tha
             client_eth_address=self.task_to_compute.provider_ethereum_address,
             pending_value=1,
         )
+
+
+class FinalizePaymentBanksterTest(ConcentIntegrationTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.task_to_compute = self._get_deserialized_task_to_compute()
+
+    def _create_transaction(self):
+        return Transaction(
+            nonce=1,
+            gasprice=10 ** 6,
+            startgas=80000,
+            value=10,
+            to=b'7917bc33eea648809c28',
+            v=28,
+            r=105276041803796697890139158600495981346175539693000174052040367753737207356915,
+            s=51455402244652678469360859593599492752947853083356495769067973718806366068077,
+            data=b'3078666333323333396130303030303030303030303030303030303030303030303032393662363963383738613734393865663463343531363436353231336564663834666334623330303030303030303030303030303030303030303030303030333431336437363161356332633362656130373531373064333239363566636161386261303533633030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303237313030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303562343336643666',
+        )
+
+    @override_settings(
+        ADDITIONAL_VERIFICATION_COST=0,
+    )
+    def test_that_when_additional_verification_cost_is_zero_finalize_payment_should_return_empty_provider_claim_payment_info(self):
+        with mock.patch('core.payments.service.get_deposit_value', return_value=1) as get_deposit_value:
+            with mock.patch('core.payments.service.force_subtask_payment', return_value=self._create_transaction()) as force_subtask_payment:
+                with mock.patch('core.payments.service.cover_additional_verification_cost', return_value=self._create_transaction()) as cover_additional_verification_cost:
+                    (requestors_claim_payment_info, providers_claim_payment_info) = finalize_payment(
+                        subtask_id=self.task_to_compute.subtask_id,
+                        concent_use_case=ConcentUseCase.ADDITIONAL_VERIFICATION,
+                        requestor_ethereum_address=self.task_to_compute.requestor_ethereum_address,
+                        provider_ethereum_address=self.task_to_compute.provider_ethereum_address,
+                        subtask_cost=1,
```

Comment:
please extract it to a field and reuse

Reply:
I don't get this one, do you mean "extract it to a constant"?

Reply:
I ment `self.subtask_cost` or sth similar ;)

Reply:
:+1: done